cfpb/consumerfinance.gov
# Clean up call to action

Follow up to https://github.com/cfpb/consumerfinance.gov/pull/7966


## Removals

- Remove unused `btn` class from call to action (buttons already center text anyway).


## Changes

- Fix button values check in call to action jinja template.
- Add aria-label option to call to action button, which inherits from hyperlink that has that option.


## How to test this PR

1. `yarn build`
2. Search for a call to action block in Wagtail and add an aria label and see it shows up in the call to action button.
